### PR TITLE
feat(ACMS-CAM-12): Implement Frontend with getAllCustomers

### PR DESCRIPTION
**Pull request by CrisWillCarry**
Jira Tickets : https://champlainsaintlambert.atlassian.net/browse/ACMS-46 , https://champlainsaintlambert.atlassian.net/browse/ACMS-39 , https://champlainsaintlambert.atlassian.net/browse/ACMS-12

**Context:**

I committed the home, log-in and sign-up pages before we implemented the PRs on GitHub because I wanted my teammates to have access to it. This PR will represent that commit as well. This is our first full slice which is for the getAllCustomers()

**Changes made:**

Implemented home page, logged in homepage, login page, sign up page and customers page. I also implemented the selenium testing for the getAllCustomers() use case. Other from that, I also fixed the back-end entity and the mappers alongside the database schema and inserts since they weren't matching and were throwing errors of unknown fields. I also inserted the 10 objects of Customer inside of the database as dummy data.

**UI/UX:

BEFORE:**

no UI

AFTER:**

footers are included for each page at the bottom.

HOME PAGE:

image
image

LOGIN PAGE:

image
image

LOGGED_IN HOME PAGE:

image
image

CUSTOMERS PAGE:

image
image

### CODE REVIEW

File: acms-react-app/src/Pages/Signup_Page/Signup.js [hunk 1]
```diff
@@ -4,27 +4,59 @@ import { faFacebook, faGoogle, faApple } from '@fortawesome/free-brands-svg-icon
...
 function Signup(){
 
-    return(
+    function signup(user){
+        axios.post("http://localhost:8080/api/v1/registration",user)
+        .then(res=>{
+            if(res.status === 201){
+                console.log("User has successfully signed up!")
+            }
+        })
```
Comment (Iantomasi) [inline acms-react-app/src/Pages/Signup_Page/Signup.js:18]: Good use of axios Cris. You made great Front-End progress for the team and it won't go unnoticed.

**Review by Iantomasi — DISMISSED** [review]
Great stuff Cris. #Rich&HandomeEnterprises

File: acms-react-app/src/Pages/Signup_Page/Signup.js (hunk 1 above)
Comment (CrisWillCarry) [reply acms-react-app/src/Pages/Signup_Page/Signup.js:18]: Thank you

**Review by Iantomasi — APPROVED** [review]
Great job Cris, this is significant progress.

**Pull request merged**